tooling/charts/tl500-base/values.yaml

```diff
@@ -78,7 +78,7 @@ logging:
 
 gitlab:
   namespace: tl500-gitlab
-  root_password: Password123
+  root_password: 7aydhn160bOrrsGEbnd172rE
```

Comment:
Can you update this in [tooling/charts/tl500-base/templates/gitlab/deployments.yaml#L64](https://github.com/rht-labs/enablement-framework/blob/main/tooling/charts/tl500-base/templates/gitlab/deployments.yaml#L64) as well?